islandora_basic_collection.module

```diff
@@ -965,3 +965,32 @@ function islandora_basic_collection_islandora_solution_pack_child_relationships(
     ];
   }
 }
+
+/**
+ * Implements hook_help().
+ */
+function islandora_basic_collection_help($route) {
+  switch ($route) {
+    // Main help page for the module.
+    case 'help.page.islandora_basic_collection':
+      // About section.
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Islandora Basic Collection module allows users to add and manage collections and their contents. See the <a href=":wiki">Islandora Collection Solution Pack wiki*</a> for more information.', [
+        ':wiki' => 'https://wiki.duraspace.org/display/ISLANDORA/Colleciton+Solution+Pack',
+      ]) .
+      '</p>';
+      // Uses section.
+      $output .= '<h3>' . t('Uses') . '</h3>';
+      $output .= '<dt>' . t('Manage Collections and Their Objects') . '</dt><dd>';
+      $output .= t('Islandora Basic Collection module can be used to perform multiple operations on collections within Islandora on your website. More information can be found on specific use cases in the "How to" links on the <a href=":usage">Getting Started section of wiki*</a>.', [
```

Comment:
Should this be "... section of the wiki"?

Reply:
Should we be linking the same thing twice so close together?

Reply:
Fixed the typo, and people tend to be lazy when reading documentation, so it's usually easier to provide direct links instead of sending them to search for a section on a previously mentioned page.